src/e2e-test/features/bigquery/source/BigQueryToBigQuery_withConnections.feature

```diff
@@ -0,0 +1,108 @@
+    Then Verify connection datatable is displayed for the data: "bqSourceTable"
+    Then Click Create Pipeline button and choose the type of pipeline as: "Batch pipeline"
+    Then Verify plugin: "BigQueryTable" node is displayed on the canvas with a timeout of 120 seconds
+    Then Navigate to the properties page of plugin: "BigQueryTable"
+    Then Verify toggle plugin property: "useConnection" is toggled to: "YES"
+    Then Verify plugin property: "connection" contains text: "bqConnectionName"
+    Then Verify input plugin property: "dataset" contains value: "dataset"
+    Then Verify input plugin property: "table" contains value: "bqSourceTable"
+    Then Validate output schema with expectedSchema "bqSourceSchema"
+    Then Validate "BigQuery" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "Wrangler"
+    Then Verify the Output Schema matches the Expected Schema: "bqSourceSchema"
+    Then Validate "Wrangler" plugin properties
+    Then Close the Plugin Properties page
+    Then Expand Plugin group in the LHS plugins list: "Sink"
+    Then Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Navigate to the properties page of plugin: "BigQuery2"
+    Then Click plugin property: "useConnection"
+    Then Click on the Browse Connections button
+    Then Select connection: "bqConnectionName"
+    Then Enter input plugin property: "referenceName" with value: "BQSinkReferenceName"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqTargetTable"
+    Then Click plugin property: "truncateTable"
+    Then Click plugin property: "updateTableSchema"
+    Then Validate "BigQuery" plugin properties
+    Then Close the Plugin Properties page
+    Then Connect plugins: "Wrangler" and "BigQuery2" to establish connection
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Validate records transferred to target table is equal to number of records from source table
+    Given Open Wrangler connections page
+    Then Expand connections of type: "BigQuery"
+    Then Open action menu for connection: "bqConnectionName" of type: "BigQuery"
+    Then Select action: "Delete" for connection: "bqConnectionName" of type: "BigQuery"
+    Then Click plugin property: "Delete" button
+    Then Verify connection: "bqConnectionName" of type: "BigQuery" is deleted successfully
+
+  @BQ_SOURCE_TEST @BQ_SINK_TEST @USE_BQ_CONNECTION
```

Comment:
nit: A better tag name could be `EXISTING_BQ_CONNECTION` instead of `USE_BQ_CONNECTION`.

Reply:
Renamed tag.